azavea/pfb-network-connectivity
# Add angularjs container and parts of an app

Connects #83.

Removes the Angular 2 stub app and replaces it with an Angular 1.4.9 app with somewhat-working user-management bits and not-currently-functional boundary upload bits.

Note that this contains Area logic that, as noted on PR #84, acts as a cross-organization grouping that might or might not be useful in some form but probably shouldn't be called Area.  I could rip it out altogether from the PR, but there are pieces in the Django app as well, and it might be simpler and more useful to have a separate issue for figuring out what we need and implementing it.

I did some work toward upgrading to Angular 1.5.11 ~~but I figured I'd put this 1.4.9 version up for now since it's hard to predict how successful that will be and how long it will take.~~ and it seemed to go fine as far as the front-end code and dependencies.  It's a pretty simple app at this point, so maybe not surprising.  Upgrading the node/build dependencies was not as clean, so I left them.

![image](https://cloud.githubusercontent.com/assets/6598836/23186056/af62c9d6-f853-11e6-88c4-f0ff4a3a4a4a.png)

**To Test**
`./scripts/update` should build the container and `./scripts/server` should run it, at which http://localhost:9301/ should get you a login page that will let you login with systems+pfb@azavea.com/root and do some user and organization management things.
